data/json/achievements.json

```
@@ -15,16 +45,40 @@
   {
     "id": "achievement_kill_10_monsters",
     "type": "achievement",
-    "name": "Decamate",
+    "name": { "str": "Decamate", "//~": "Pun on 'deca-' prefix for 10 and 'decimate', with it's meaning for killing" },
```

Does this work? I see support for `ctxt` in `translation::deserialize`, but nothing for `//~`
```suggestion
    "name": { "str": "Decamate", "ctxt": "Pun on 'deca-' prefix for 10 and 'decimate', with it's meaning for killing" },
```

---

It's not in `translation::deserialize` because it's not needed there.  It is handled in `lang/extract_json_strings.py`, so I think it should work (haven't actually tested).

---

Ah, of course, my bad.